analysis_driver/dataset.py

```diff
@@ -723,15 +730,12 @@ def species(self):
     @property
     def genome_version(self):
         if self._genome_version is None:
-            g = clarity.get_sample(self.samples_processed[0]['sample_id']).udf.get('Genome Version')
-            if not g:
-                g = rest_communication.get_document('species', where={'name': self.species})['default_version']
-            self._genome_version = g
+            self._genome_version = self.sample_dataset(self.samples_processed[0]['sample_id']).genome_version
```

Comment:
Can you please point me to where this information is now downloaded, since the "get_document" call has been removed?

Reply:
The logic here is identical to SampleDataset.genome_version at line 500, so ProjectDataset.genome_version just goes through that now

Reply:
The logic here is identical to SampleDataset.genome_version at line 500, so ProjectDataset.genome_version just goes through that now